### PR TITLE
Remove deprecated methods in Cart class

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Removes some deprecated methods in Cart class.
| Type?             | refacto
| Category?         | CO
| BC breaks?        | yes
| Deprecations?     | no
| How to test?      | CI and tests green - https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/5556922543
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   | 

### Removed methods
- Cart::containsProduct
- Cart::simulateCarriersOutput

### Deprecated methods
- Cart::simulateCarrierSelectedOutput
- Cart::intifier
- Cart::desintifier

### CODE REVIEW

**Review by FabienPapet — DISMISSED** [review]

File: classes/Cart.php [hunk 1]
```diff
@@ -3099,116 +3069,6 @@ public function carrierIsSelected($id_carrier, $id_address)
...
-    }
-
-    /**
-     * Simulate output of selected Carrier.
-     *
-     * @param bool $use_cache Use cache
-     *
-     * @return int Intified Cart output
-     */
-    public function simulateCarrierSelectedOutput($use_cache = true)
```
Comment (leemyongpakvn) [inline classes/Cart.php:3171]: 2 functions Cart::containsProduct and Cart::simulateCarriersOutput are clearly deprecated but I'm a bit worry about 3 others
Cart::simulateCarrierSelectedOutput
Cart::intifier
Cart::desintifier

Comment (kpodemski) [reply classes/Cart.php:3171]: If we want to delete them, we have to deprecate them first.

**Review by kpodemski — CHANGES_REQUESTED** [review]
You removed some of the methods that were not previously deprecated :D

File: classes/Cart.php (hunk 1 above)
Comment (Hlavtox) [reply classes/Cart.php:3171]: Done, deprecated only

**Comment by Hlavtox** [comment]
@kpodemski @leemyongpakvn ready :-)

**Review by leemyongpakvn — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-automation do you know if failed auto-tests are known issues?

**Review by FabienPapet — APPROVED** [review]

**Comment by Progi1984** [comment]
> @PrestaShop/qa-automation do you know if failed auto-tests are known issues?

There are known issues.

**Pull request merged**